docs/core/tools/telemetry.md

```diff
@@ -42,16 +42,14 @@ The .NET Core CLI Tools telemetry feature is enabled by default. Opt-out of the
 
 The feature collects the following data:
 
-- Timestamp&#8224;
-- Command (for example, "build")&#8224;
+- Timestamp of invocation&#8224;
+- Command invoked (for example, "build")&#8224;
 - Geographical location&#8224;
 - `ExitCode` of the command
 - Test runner (for test projects)
 - Operating system and version&#8224;
-- Runtime&#8224;
-- CLI (SDK) version&#8224;
-- CLI installation success/failure
-
+- Whether runtime IDs are present in the runtimes node&#8224;
```

Comment:
I believe it reports the actual runtime (at least from looking at the data). It doesn't report anything like true/false "there are runtimes present in the runtimes node," so that that's why I changed this from this line (that you reverted) to just "Runtime."